policy/cell_geometry.yaml

```diff
@@ -0,0 +1,86 @@
+---
```

Comment:
Those are constants. Changing those coordinates requires modifyng the mirror cell. It's better and permissible to leave those as part of the code, e.g. see python/lsst/ts/cRIOpy/M1M3FATable.py in ts_cRIOpy. There aren't any real benefits to have this configurable, you will just need to somehow handle correct path - you are entering too many problems for little gains. Not to mention PySide2 has its own configuration system. I will prefer that as .py inside code.